SomerenUI/EditStudentForm.cs

```diff
@@ -8,45 +7,34 @@ namespace SomerenUI
     public partial class EditStudentForm : BaseForm
     {
         Student currentStudent;
+        const int FormWidth = 411;
+        const int CreateStudentHeight = 486;
+        const int EditStudentHeight = 350;
```

Comment:
![image](https://github.com/AnastasiaIU/inholland-year1-term3-project-database/assets/69650704/f843cf06-75c3-4551-89e6-3eae7f83ea47)

![image](https://github.com/AnastasiaIU/inholland-year1-term3-project-database/assets/69650704/c484b8ef-5d87-456f-9f04-d7fea651b422)

This is how it looks on my screen with 100% scale. Are you sure you want to use these sizes?

Reply:
Looks nice on my screen, only I suggest increasing the sizes for the Create/Update buttons.
![image](https://github.com/AnastasiaIU/inholland-year1-term3-project-database/assets/69650704/d52db5f4-29a8-4d13-8454-c317a63dd381)